sources/packages/web/src/composables/useFormioDropdownLoader.ts

```diff
@@ -16,15 +19,17 @@ export function useFormioDropdownLoader() {
   const loadDropdown = async (
     form: any,
     dropdownName: string,
-    loadMethod: Promise<OptionItemAPIOutDTO[]>,
+    loadMethod: Promise<OptionItemAPIOutDTO[]> | ProgramIntensityAPIOutDTO[],
```

Comment:
I would prefer defining `id: number | string` instead of creating new object

Reply:
I agree with @dheepak-aot, please avoid adding the `ProgramIntensityAPIOutDTO`. The `OptionItemAPIOutDTO` was created to be generic and the objects must be adapted to it because `OptionItemAPIOutDTO` has what is needed to populate a dropdown.